x/profile/client/rest/tx.go

```diff
@@ -0,0 +1,114 @@
+package rest
+
+import (
+	"net/http"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
+	"github.com/desmos-labs/desmos/x/profile/internal/types"
+	"github.com/gorilla/mux"
+)
+
+func registerTxRoutes(cliCtx context.CLIContext, r *mux.Router) {
+	r.HandleFunc("/profile/create", createProfileHandler(cliCtx)).Methods("POST")
+	r.HandleFunc("/profile/edit", editProfileHandler(cliCtx)).Methods("PUT")
+	r.HandleFunc("/profile/delete", deleteProfileHandler(cliCtx)).Methods("DELETE")
```

Comment:
I think it might be better to change these methods to include the address of the profile owner such as: 

```suggestion
	r.HandleFunc("/profiles/{address}", editProfileHandler(cliCtx)).Methods("PUT")
	r.HandleFunc("/profiles/{address}", deleteProfileHandler(cliCtx)).Methods("DELETE")
```

This way the user does not have to specify it inside the request.

Reply:
does the same endpoint for both edit and delete works?

Reply:
@bragaz Yes, it would work as the `Methods("DELETE")` and `.Methods("PUT")` will identify them uniquely. This means that the endpoint will be the same, but the action taken will be based on the HTTP method used to contact such endpoint.